src/components/GitHubActivityMap.js

```diff
@@ -1,16 +1,99 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './GitHubActivityMap.css';
+import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet'
+const dotEnv = require('dotenv').config();
+
 
 const GitHubActivityMap = (props) => {
 
-  //d3 and leaflet stuff
+  const [events, setEvents] = useState([]);
+  const [locations, setLocations] = useState([]);
+  const [markers, setMarkers] = useState([]);
+  const [ghError, setGhError] = useState('');
+  const [individualError, setIndividualError] = useState('');
+  const [geoErrors, setGeoErrors] = useState([]);
```

Comment:
way to be meticulous about error tracking -- we'll have to think about whether these different error types will actually be displayed differently. If not, we could probably store them on one state property that impacts DOM